cpp/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.14..3.16)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+project(LiftKata VERSION 1.0
+        DESCRIPTION "The lift kata for an approval testing approach"
+        LANGUAGES CXX)
+include(FetchContent)
+
+FetchContent_Declare(
+        catch2
+        GIT_REPOSITORY https://github.com/catchorg/Catch2.git
+        GIT_TAG v2.9.2
+)
+FetchContent_MakeAvailable(catch2)
+LIST(APPEND CMAKE_MODULE_PATH
+        ${catch2_SOURCE_DIR}/contrib
+        )
+
+FetchContent_Declare(
```

Comment:
In case they are hidden because the file is so large.... Please see my comments against ` cpp/include/ApprovalTests.v.7.0.0.hpp` - I'm unclear which of the two ways of accessing ApprovalTests.cpp needs is intended to be used... Thanks!